### ISSUE TITLE
[Feature] Auto Load DLC and Latest Update

### ISSUE THREAD

**Issue by amurgshere** [ryujinx-mirror/ryujinx#55]
The ability to auto load all compatible DLC and the latest update from a specified directory or list of directories automatically would be a great feature. I found this commit here on Greemdays fork which accomplishes this, it would be awesome if someone could pull these changes in:

https://github.com/GreemDev/Ryujinx/commit/565acec46850fc66def70adb0a9fae94f9ed8e59

I will do this and test when I get a chance, but created this issue in the meantime in case anyone else has time to do it before me.

**Comment by darkuser89** [comment]
why don't we all work together on a project again?
everyone cooks their own soup!

**Comment by extherian** [comment]
Nothing stopping amurgshere, vudjun and the others from submitting pull requests to both projects if they feel like it. 

On topic, amurgshere feel free to just submit the pull request if you like, regginator has been having more real-life issues that will delay him, so he won't be doing anything for the next couple of days.

**Comment by Maskworkr** [comment]
as long as we get the same level of development on forks should be good already when the time comes, well have backup repos
with updates

**Comment by GreemDev** [comment]
I'd just like to mention that the Autoloading functionality being merged into main Ryujinx was awaiting removal of the GTK UI, because it just didn't work properly with this addition seemingly. I dropped the GTK UI as I way prefer the Avalonia one anyways and asked J-Swift to PR the change because I knew it was just waiting on that being removed.

**Comment by amurgshere** [comment]
Thanks @GreemDev , I agree that the GTK UI isn't the best, happy to wait and see what @regginator  wants to do with this, we can either add the UI features to the GTK UI somehow, remove the GTK like you have, or not include the Auto DLC / Updates functionality yet.

**Comment by regginator** [comment]
> Thanks @GreemDev , I agree that the GTK UI isn't the best, happy to wait and see what @regginator  wants to do with this, we can either add the UI features to the GTK UI somehow, remove the GTK like you have, or not include the Auto DLC / Updates functionality yet.

Yea, I'm down with removing the Gtk3 UI impl entirely at some point going forward (as well as the infrastructure for it of course), as the Ava impl is great.
